### DISCUSSION TITLE
/products/pricing/v0/listings/{SellerSKU}/offers cannot contain +

### DISCUSSION THREAD

**Discussion by rokey001** [amzn/selling-partner-api-models#3174] [General]
https://sellingpartnerapi-eu.amazon.com/products/pricing/v0/listings/FLSM-GYBJ3801%2B09%2B03%2B4044-SNR-DECX1/offers

{ "errors": [ { "code": "InvalidInput", "message": "FLSM-GYBJ3801 09 03 4044-SNR-DECX1 is an invalid SKU for marketplace A1PA6795UKMFR9", "details": "" } ] }Array ( [errors] => Array ( [0] => Array ( [code] => InvalidInput [message] => FLSM-GYBJ3801 09 03 4044-SNR-DECX1 is an invalid SKU for marketplace A1PA6795UKMFR9 [details] => ) ) )

Seller_SKU : FLSM-GYBJ3801+09+03+4044-SNR-DECX1

**Comment by weswilliams-channeladvisor** [comment]
We ran into this last week and Amazon released a fix.  Make sure you are escaping the path in your request and also you might need to update your signing code.
